src/cat.jl

```diff
@@ -0,0 +1,24 @@
+function equalvalued(X::NTuple)
+    n = length(X)
+    allequal = true
+    i = 2
+    while allequal && i <= n
+        allequal = X[i] == X[i-1]
+        i += 1
+    end #while
+    return allequal
+end #equalvalued
+
+sizes{T<:AxisArray}(As::T...) = tuple(zip(map(size, As)...)...)
+matchingdims{N,T<:AxisArray}(As::NTuple{N,T}) = all(equalvalued, sizes(As...))
+matchingdimsexcept{N,T<:AxisArray}(As::NTuple{N,T}, n::Int) = all(equalvalued, sizes(As[[1:n-1; n+1:end]]...))
+
+Base.cat{T<:AxisArray}(n::Int, As::T...) = if n <= ndims(As[1])
```

Comment:
Fixed in https://github.com/mbauman/AxisArrays.jl/pull/26/commits/d5b9d24d743916af432b68df3ddf497b097ccd93